docs/src/pages/index.soy

```diff
@@ -48,7 +48,7 @@ description: "A collection of well known Algebraic Data Types for your utter enj
       <div class="row">
         <div class="col-md-12 col-md-offset-2">
           <h3 class="about-title">Why?</h3>
-          <p class="about-description"> Functional Programming matters and Crocks makes that world simplier by both curating and providing not only a common interface between each type (where possible of course), but also all of the helper functions needed to hit the ground running.</p>
+          <p class="about-description">Because Functional Programming! Crocks adds more tools to your belt by curating a common interface between each data type (where possible of course), and including all of the helper functions needed to hit the ground running.</p>
```

Comment:
Do not know about this change either, crocks curates the types not the interface, that is defined by the Fantasy Land Spec. The purpose of crocks is so a user of the lib does not have to maintain their own implementation of common types.

So it does not add tools to their belt, it keeps them from implementing those "tools"

Reply:
> crocks curates the types not the interface, that is defined by the Fantasy Land Spec

You wrote this, I edited to make it more readable

> The purpose of crocks is so a user of the lib does not have to maintain their own implementation of common types

That's a given for any lib... what are we doing here?